instrumentation/mysql2/lib/opentelemetry/instrumentation/mysql2/patches/client.rb

```diff
@@ -65,7 +63,7 @@ def query(sql, options = {})
           private
 
           def obfuscate_sql(sql)
-            return sql unless config[:enable_sql_obfuscation]
+            return sql if config[:db_statement] == :include
```

Comment:
Now that there is a check on L56, is this check redundant?

Reply:
🤔  I think the check on line 56, if its include or obfuscate it would call `obfuscate_sql(sql)` which will check the flag again. In this state its a necessary check. However I see how checking the same flag twice is a code smell.

Reply:
Ill try to refactor, see if I can do better